redcap_bridge/tests/test_utils.py

```diff
@@ -13,3 +13,9 @@ def test_compressedCSV(initialize_test_dir):
             exp = exp_file.read()
             assert res == exp
 
+
+def test_conversion_to_odml_table_descriptor(initialize_test_dir):
+    test_dir = test_directory / 'testfiles_redcap' / 'descriptors'
+
+    conversion_to_odml_table_descriptor(test_dir / 'Vision4Action_DATA_2023-04-13_1110.csv', session_number=5)
```

Comment:
this test is only checking if the code runs into an error so far. Could you extend this and also test the content of the converted file? Maybe you will need to add an odmltables reference test file to compare against.

Reply:
Yes, will change the test in priority